indico/modules/events/abstracts/templates/display/abstracts.html

```diff
@@ -0,0 +1,77 @@
+{% extends 'layout/conference_page_base.html' %}
+{% from 'events/abstracts/management/_abstract_list.html' import render_abstract_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
```

Comment:
IMO it's not that important to know exactly what it `extends` from, if you wanna know, you can look for it in the first lines. I would prefer to see the `import`s first and then the `extends`. The `import`s are not affected by the `extend`, but the rest of the template (`macro`s) are based on the `extend`, so I would keep those closer and the `import`s at the top.